swift/Samples/Tutorial/Tutorial5.md

```diff
@@ -0,0 +1,838 @@
+# Step 5
+
+_Unit and Integration Testing Workflows_
+
+## Setup
+
+Start from the completed code after Tutorial 4, or if you're jumping ahead, the completed `Tutorial4` pod. The default scheme of `Tutorial` will run unit tests from `TutorialBase` and `Tutorial4` when pressing `cmd+shift+u`
```

Comment:
I would leave explicit instructions in here: "to follow this tutorial, open your terminal and do X to get Xcode into a buildable state".

It's common to jump straight to a tutorial when it's relevant, so it would be nice to make this as easy as possible.

Reply:
Updated in this tutorial as well as all the previous ones. Good call.